auto_invitation_to_notion/main.py

```diff
@@ -0,0 +1,127 @@
+import requests
+import copy
+from collections import defaultdict
+from typing import List, Dict
+
+from notion.client import NotionClient
+
+
+LUBYCON_PRIVATE_PAGE = '4594ead1-b975-451d-a578-2ffeea6aa452'
+LUBYCON_MENTOR_PAGE = '720e43a5-f572-440a-8e58-847b72359b16'
+LUBYCON_HUB_PAGE = 'd3ebe34b-34af-4b2f-b984-ae1e91cff7f3'
+
+
+def get_lubycon_admin_uid(client: NotionClient) -> str:
+    return client.get_email_uid().get('lubycon@gmail.com')
+
+
+def get_workspace_id(client: NotionClient, admin_uid: str) -> str:
+    response = client.post('getSpaces', {})
+    if response.ok:
+        workspace_id = list(response.json()[admin_uid]['space'].keys())[-1]
+        return workspace_id
+    else:
+        raise Exception(handle_request_error(response=response))
+
+
+def handle_request_error(response: requests.Response) -> None:
+    raise Exception(
+        f"request fail!. status code: {response.status_code}, detail: {response.text}")
+
+
+def get_uid_pageids_assigned_users(client: NotionClient, workspace_id: str) -> Dict:
+    response = client.post("getSubscriptionData", {
+                           "spaceId": workspace_id, "version": "v2"})
+    if response.ok:
+        users_info = response.json()["users"]
+
+        answer = {}
+        for user_info in users_info:
+            answer[user_info["userId"]] = {
+                "guest_page_ids": [] if user_info.get("guestPageIds") is None else user_info.get("guestPageIds")}
```

Comment:
```suggestion
                "guest_page_ids": user_info.get("guestPageIds") or []}
```
처럼 `or` 연산자로 Nullish Coalescing 연산자의 효과를 노려볼 수도 있을 것 같습니다. 좌변의 값이 `0` 같은 Fasly Value여도 우변의 값이 할당되버린다는 점이 슬프기는 하지만... `guestPageIds` 필드의 값이 Number는 아닌 것 같아서 괜찮을 것 같아유

Reply:
Lubycon admin의 경우에는 `guestPageIds`라는 필드값이 없습니다. 
나머지 사용자들은 모두 리스트로 가지고있고요. 따라서 Fasly Value는 크게 걱정하지 않아도 될 것 같아요.
`or`로 하는게 조금 더 라인이 깔끔해질 것 같아서 그렇게 반영하도록 할게요.
감사합니다.

Reply:
반영 완료했습니다.

Reply:
https://www.tutorialspoint.com/python/dictionary_get.htm
user_info.get("guestPageIds", []) 하면 되긴합니다 ㅋㅋㅋ